store/tikv/coprocessor.go

```diff
@@ -78,7 +78,7 @@ func (c *CopClient) supportExpr(exprType tipb.ExprType) bool {
 	case tipb.ExprType_Case, tipb.ExprType_If, tipb.ExprType_IfNull, tipb.ExprType_Coalesce:
 		return true
 	// aggregate functions.
-	case tipb.ExprType_Count, tipb.ExprType_First, tipb.ExprType_Max, tipb.ExprType_Min, tipb.ExprType_Sum, tipb.ExprType_Avg:
+	case tipb.ExprType_Count, tipb.ExprType_First, tipb.ExprType_Max, tipb.ExprType_Min, tipb.ExprType_Sum, tipb.ExprType_Avg, tipb.ExprType_Agg_BitOr:
```

Comment:
TiKV doesn't support ExprType_Agg_BitOr.

Reply:
@shenli how about this:
* Step1:  disable `FinalMode` for `BIT_OR` so that it's not calculated in TIKV.
* Step2: take another pr in TiKV to support `BIT_OR` IN TIKV.
* Step3: re-enable `FinalMode` for `BIT_OR`

Reply:
If the agg is bit_or, we return `c.store.mock` so as to push it to mock tikv instead of real tikv.

Reply:
@hanfei1991 
1. you mean that `BIT_OR` is never push down to `TiKV`, even in real clustered environment? 
2. This pr seems works ok with mock-tikv
3. I think it's still necessary to support `BIT_OR` in TiKV anyway
what do you think

Reply:
After tikv has supported this aggregate function, we can remove this check. This is only a protection mechanism.

Reply:
ok

Reply:
@hanfei1991 PTAL. I disabled `FinalMode` for  aggregation `BIT_OR`. This check seems can be removed